.ci.yaml

```diff
@@ -2764,7 +2764,6 @@ targets:
 
   - name: Mac plugin_lint_mac
     recipe: devicelab/devicelab_drone
-    bringup: true # Flaky https://github.com/flutter/flutter/issues/112146
```

Comment:
Make this run in presubmit.

Reply:
Do you want this as a temp presubmit check validation (seems it passes) or want to remove the flake flag as it is in this PR? Wondering if you want to have this target to go through the flake validation workflow.

Reply:
I was going to revert it in this PR in case this didn't solve the flake.  Just first wanted it to run in try, which I think it won't if bringup is true?
It passed:
https://ci.chromium.org/p/flutter/builders/try/Mac%20plugin_lint_mac/18408

Reply:
Reverted.